framework/src/modules/chain/logic/state_store/account_store.js

```diff
@@ -0,0 +1,137 @@
+const _ = require('lodash');
+
+const defaultAccount = {
+	publicKey: null,
+	secondPublicKey: null,
+	secondSignature: false,
+	u_secondSignature: false,
+	username: null,
+	u_username: null,
+	isDelegate: false,
+	u_isDelegate: false,
+	balance: '0',
+	u_balance: '0',
+	missedBlocks: 0,
+	producedBlocks: 0,
+	rank: null,
+	fees: '0',
+	rewards: '0',
+	vote: '0',
+	nameExist: false,
+	u_nameExist: false,
+	multiMin: 0,
+	u_multiMin: 0,
+	multiLifetime: 0,
+	u_multiLifetime: 0,
+};
+
+class AccountStore {
+	constructor(accountEntity, { mutate } = { mutate: true }) {
+		this.account = accountEntity;
```

Comment:
wouldn't it be better to name this property `accountEntity` so when reading other methods it's very clear that the operation it's an storage access?

Reply:
There is no other property named `account` in `AccountStore`. So, I think it is fine. If you think it is really confusing then i can change it. :)

Reply:
No it's ok you have a good point :)